src/Service/Indexer/SiteKit/DefaultSchema2xDocumentEnricher.php

```diff
@@ -0,0 +1,457 @@
+            return '';
+        }
+
+        $content = [];
+        foreach (($contactPoint['contactData']['phoneList'] ?? []) as $phone) {
+            $countryCode = $phone['phone']['countryCode'] ?? '';
+            if (
+                !empty($countryCode) &&
+                !in_array($countryCode, $content, true)
+            ) {
+                $content[] = '+' . $countryCode;
+            }
+            $areaCode = $phone['phone']['areaCode'] ?? '';
+            if (!empty($areaCode) && !in_array($areaCode, $content, true)) {
+                $content[] = $areaCode;
+                $content[] = '0' . $areaCode;
+            }
+            $content[] = ($phone['phone']['localNumber'] ?? '');
+        }
+        foreach (($contactPoint['contactData']['emailList'] ?? []) as $email) {
+            $content[] = $email['email'];
+        }
+
+        if (isset($contactPoint['addressData'])) {
+            $addressData = $contactPoint['addressData'];
+            $content[] = ($addressData['street'] ?? '');
+            $content[] = ($addressData['buildingName'] ?? '');
+            $content[] = (
+                $addressData['postOfficeBoxData']['buildingName'] ?? ''
+            );
+        }
+
+        return implode(' ', $content);
+    }
+
+    private function idWithoutSignature(string $id): int
+    {
+        $s = substr($id, -11);
+        return (int)$s;
+    }
+
+    /* Customization
+     * - https://gitlab.sitepark.com/customer-projects/fhdo/blob/develop/fhdo-module/src/publish/php/SP/Fhdo/Component/Content/DetailPage/StartletterIndexSupplier.php#L31
+     * - https://gitlab.sitepark.com/apis/sitekit-php/blob/develop/php/SP/SiteKit/Component/Content/NewsdeskRss.php#L235
+     * - https://gitlab.sitepark.com/customer-projects/fhdo/blob/develop/fhdo-module/src/publish/php/SP/Fhdo/Component/SearchMetadataExtension.php#L41
+     * - https://gitlab.sitepark.com/customer-projects/paderborn/blob/develop/paderborn-module/src/publish/php/SP/Paderborn/Component/FscEntity.php#L67
+     * - https://gitlab.sitepark.com/customer-projects/paderborn/blob/develop/paderborn-module/src/publish/php/SP/Paderborn/Component/FscContactPerson.php#L24
+     * - https://gitlab.sitepark.com/customer-projects/stadtundland/blob/develop/stadtundland-module/src/publish/php/SP/Stadtundland/Component/ParkingSpaceExpose.php#L38
+     * - https://gitlab.sitepark.com/customer-projects/stadtundland/blob/develop/stadtundland-module/src/publish/php/SP/Stadtundland/Component/Expose.php#L38
+     * - https://gitlab.sitepark.com/customer-projects/stadtundland/blob/develop/stadtundland-module/src/publish/php/SP/Stadtundland/Component/PurchaseExpose.php#L38
+     * - https://gitlab.sitepark.com/ies-modules/citycall/blob/develop/citycall-module/src/main/php/src/SP/CityCall/Component/Intro.php#L51
+     * - https://gitlab.sitepark.com/ies-modules/sitekit-real-estate/blob/develop/src/publish/php/SP/RealEstate/Component/Expose.php#L47
+     */
```

Comment:
maybe save these in a readme or some notes or at least a class comment; here is a bit random